src/freenet/store/saltedhash/CipherManager.java

```diff
@@ -55,12 +57,14 @@ byte[] getDiskSalt() {
 	 * Cache for digested keys
 	 */
 	@SuppressWarnings("serial")
-	private Map<ByteArrayWrapper, byte[]> digestRoutingKeyCache = new LinkedHashMap<ByteArrayWrapper, byte[]>() {
+	final private Map<ByteArrayWrapper, byte[]> digestRoutingKeyCache = new LinkedHashMap<ByteArrayWrapper, byte[]>() {
```

Comment:
Sorry, gotta nitpick: according to [JLS §8.3.1](https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.3.1) `private` comes before `final`. 🙂